.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+name: CI
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clean
+        uses: easimon/maximize-build-space@v4
+        with:
+          root-reserve-mb: 20480
+          swap-size-mb: 1024
+          remove-dotnet: 'true'
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        run: |
+          sudo apt update
+          sudo apt install -y ccache flatpak flatpak-builder xvfb zstd
+          sudo rm -rf /var/lib/{apt,dpkg,cache,log}/
+
+          sudo flatpak remote-add --system --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: ./.flatpak-builder
+          key: flatpak-builder
```

Comment:
I belive that https://github.com/elementary/calculator/pull/169#discussion_r615700732 is true here too.

Reply:
Updated